templates/scripts/pre-k8s.sh

```diff
@@ -19,6 +20,18 @@ if [[ $? -ne 0 ]]; then
     fi
 fi
 
+<% if ne (index .Params `sendgridApiKey`) "" %>
```

Comment:
oh i just realized this part, we also created it in https://github.com/commitdev/zero-aws-eks-stack/blob/main/templates/terraform/bootstrap/secrets/main.tf#L36-L45

Reply:
Yeah, oops. I'll pull this out.